scripts/script.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
At some point, it might become hard to maintain / bring fixes or refactoring from the official node & squad installation scripts into this one (e.g. if we find out that an alternative pattern for say, cloning repositories or setting up systemd units).

Maybe we can think of this software as a plugin, and implement a sort of plugin system in the main installation scripts?

On the other hand, they can be seen as independent software, so having these scripts with a different lifetime is also all right :+1: